SampleDanger/ViewController.swift

```diff
@@ -649,6 +649,647 @@ class ViewController: UIViewController {
                                          
     // Do any additional setup after loading the view.
   }
+  
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 652;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)